app/controllers/account_two_factors_controller.rb

```diff
@@ -49,6 +49,11 @@ def new_2fa?
   end
   helper_method :new_2fa?
 
+  def remove_token
```

Comment:
Let's use `2fa_memory` or `two_factor_memory` instead of `token`.  I'm concerned this will get confusing since the code and backup codes are kind of like a "token."  We should change this throughout.